mappings/net/minecraft/server/dedicated/MinecraftDedicatedServer.mapping

```diff
@@ -1,7 +1,12 @@
 CLASS tr net/minecraft/server/dedicated/MinecraftDedicatedServer
 	FIELD g LOGGER_DEDICATED Lorg/apache/logging/log4j/Logger;
+	FIELD h sha1Pattern Ljava/util/regex/Pattern;
```

Comment:
If this is static final (I'm not sure if it is), it should be WRITTEN_THIS_WAY

Reply:
It is a static final field.

Reply:
Oups, I didn't notice!